examples/basic/chat-search.py

```diff
@@ -17,6 +17,7 @@
 
 from langroid.agent.chat_agent import ChatAgent, ChatAgentConfig
 from langroid.agent.task import Task
+from langroid.agent.tools.sciphi_search_rag_tool import SciPhiSearchRAGTool
```

Comment:
I think you forgot to include the file that has this tool

Reply:
yes, apologies. Adding this and an example use